__tests__/ExpensiMark-HTML-test.js

```diff
@@ -1959,5 +1959,92 @@ describe('Image markdown conversion to html tag', () => {
         const testString = '![`code`](https://example.com/image.png)';
         const resultString = '<img src="https://example.com/image.png" alt="<code>code</code>" />';
         expect(parser.replace(testString)).toBe(resultString);
-    })
+    });
+});
+
+describe('room mentions', () => {
```

Comment:
I think we can add a test for:
- `#` without following characters, should yield no mention
- Room name with numbers
- Room name with hyphen